### PR TITLE
build to firefox

### PR DESCRIPTION
**Pull request by JPedroBorges**
For compatibility reasons, Firefox accepts the `chrome` keyword, so no change is needed to the extension.
The extension is currently under review by Firefox, if not accepted we may need to revisit the build process.

Added a NPM script to build with Firefox.

```bash
npm run buildFirefox
npm run packageFirefox
```

All the specific build requirements for either chrome or Firefox browsers and declared in the webpack specific script.
- webpack.chrome.js
- webpack.firefox.js

The commonalities between builds will be in
- webpack.common.js

### CODE REVIEW

File: webpack.firefox.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            return modify(content).toString().replace("src/popup.html", "popup.html");
+          },
+        },
+      ],
+    }),
+  ],
+});
+
+function modify(buffer) {
+  let manifest = JSON.parse(buffer.toString());
```
Comment (Marianarodrigues7) [inline webpack.firefox.js:25]:
```suggestion
  const manifest = JSON.parse(buffer.toString());
```

Comment (JPedroBorges) [reply webpack.firefox.js]: thanks, done 👍🏻

**Review by jpmborges — COMMENTED** [review]
test from another account

**Review by bmg13 — APPROVED** [review]

**Review by Marianarodrigues7 — APPROVED** [review]
GJ 🥇

**Pull request merged**